stack/stack.py

```diff
@@ -3,10 +3,7 @@ def __init__(self):
         self._list = []
 
     def __str__(self):
-        return str(self._list)
-
-    def __len__(self):
-        return len(self._list)
+        raise Exception("Not allowed to view stack contents")
```

Comment:
You may simply remove this function from here. Throwing an exception on converting to string seems inappropriate.